### PR TITLE
Replace 2017 rulebook with 2018 one

### PR DESCRIPTION
**Pull request by kyordhel**
All changes required, such as:
- Year update
- Version update (draft)
- Readme.md
- TC list

### CODE REVIEW

**Comment by balkce** [comment]
I believe the new official Facebook Page is: https://www.facebook.com/robocup.org/

File: README.md [hunk 1]
```diff
@@ -41,7 +40,8 @@ Contributions and improvements to the rulebook are always welcome in the form of
 
 ### More information
 - @Home Website: [http://robocupathome.org/](http://robocupathome.org/)
-- RoboCup 2017 Website [http://robocup2017.org/](http://robocup2017.org/)
-- RoboCup @Home Wiki [http://wiki.robocup.org/@Home_League](http://wiki.robocup.org/@Home_League)
+- RoboCup 2018 Website [http://robocup2018.org/](http://robocup2017.org/)
```
Comment (LoyVanBeek) [inline README.md]: Inconsistent

File: pages/acknowledgments.tex [hunk 2]
```diff
@@ -36,37 +38,42 @@ \section*{Acknowledgments}
...
 
 We also like to thank all the people who contributed to the RoboCup@Home league with their feedback and comments. 
 
+% MAURICIO MATAMOROS @2018: Dummy names. Please addd the contributor username after the @
 ~\\\noindent People that have been working on this rulebook as member the league (in alphabetical order):
 \begin{multicols}{3}%
 \noindent%
-Yuki Furuta\\%                     @furushchev
-Noe Hernandez Sanchez\\%           @nohernan
+Erika Mustermann\\%                @ Name place-holder (German)
```
Comment (LoyVanBeek) [inline pages/acknowledgments.tex]: Can we outcomment these placeholders, to avoid confusion?

File: README.md (hunk 1 above)
Comment (kyordhel) [inline README.md]: This one is fixed.

File: pages/acknowledgments.tex (hunk 2 above)
Comment (kyordhel) [inline pages/acknowledgments.tex]: K. I'm not sure if I'm following you but I commented out everything. In any case I'll modify it when the final version is about to be released.

**Pull request merged**